Prose/ProseLib/Sources/ProseCoreViews/API/MessagingContext.swift

```diff
@@ -0,0 +1,18 @@
+//
+// This file is part of prose-app-macos.
+// Copyright (c) 2022 Prose Foundation
+//
+
+import Foundation
+import ProseCoreTCA
+
+public struct MessagingContext {
+  public let setAccountJID: JSFunc1<JID, Void>
```

Comment:
This is because the JID encodes as `"{\"node\":\"preview\",\"domain\":\"prose.org\"}"`, as it's not mapped to `.jidString` anymore. And by the way, `.jidString` isn't even correct, as it should be `.bareJid.rawValue`, but it didn't exist so I left `.jidString`.

Reply:
Sorry about that. Fixed. I've also added a test that check how the JID is encoded.